.circleci/src/jobs/@create-audius-app-jobs.yml

```diff
@@ -36,3 +36,30 @@ create-audius-app-test:
         name: 'test'
         command: |
           npx turbo run test --filter=create-audius-app
+
+create-audius-app-test-e2e:
+  working_directory: ~/audius-protocol
+  resource_class: medium
+  executor: cypress/default
+  steps:
+    - checkout
+    - attach_workspace:
+        at: ./
+    - run:
+        name: 'test-e2e'
+        command: |
+          npx playwright install --with-deps
+          cd packages/create-audius-app
+          npm run setup-test-env
+          cd examples/react
+          npm run build
+          npm run start
```

Comment:
very minor nit: you could put some of this into your playwright config, similar to what Marcus did here
https://github.com/AudiusProject/audius-protocol/pull/7922/files#diff-c8f459e3e20f9b5b0e605d427e22023ae1f23be6900db4e4988aea0603794e95R102

Reply:
Cool, I was considering that but wanted to see if I could make it work without needing a config file. I still have to fix CI so may end up adding that though!